mixer/adapter/stackdriver/helper/common.go

```diff
@@ -19,9 +19,15 @@ import (
 
 	gapiopts "google.golang.org/api/option"
 
```

Comment:
style nit: i think this requires a fmt pass, as there should only be 3 import blocks.